### DISCUSSION TITLE
Welcome to pitweetpay-integration Discussions!

### DISCUSSION THREAD

**Discussion by KOSASIH** [KOSASIH/pitweetpay-integration#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ahmedfemo** [comment]
Dear kosasih

    After greating 
We are pi network community and we need some answers for the codes belong to pi network apps

1. Are the open mainnet will be lunch on 1 june 2024 as you wrote?

2. Are the pi coin will be listed on exchanges on 1 june2024?

3. Are the start price will be 314.159 usd 
4. Are you one of the developers belong to the core team?

Kindly we all pi community need answer for that 

You can answer by the correct answer.
Or answer with i can’t answer if that is secret. 

We only need answers
